repositories/lib/services/entity.repository.js

```diff
@@ -54,6 +54,15 @@ class EntityRepository {
   async update(entity, trx) {
     //Keep the input parameter immutable
     const entityDto = _.cloneDeep(entity);
+
+    const identityClause = {};
+
+    if (_.isArray(this.idColumn)) {
```

Comment:
considering that this just calls Array.isArray under the hood, using just that should be sufficient in modern Node :)